spec/daisybill_api/models/patient_spec.rb

```diff
@@ -7,7 +7,7 @@
   it_behaves_like DaisybillApi::Ext::Attributes::SendAs, :first_name, :last_name, :gender, :date_of_birth,
     :ssn, :address_attributes, :suffix, :middle_initial, :practice_internal_id, :telephone,
     :billing_provider_id
-  it_behaves_like DaisybillApi::Ext::CRUD, :all, :find, :create, :update, :destroy, '/patients',
+  it_behaves_like DaisybillApi::Ext::CRUD, :all, :find, :create, :update, :destroy, :search, '/patients',
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.